### PR TITLE
Add new environment `evaluation`

### PR DESCRIPTION
**Pull request by iamrk04**
- add new env for aoai evaluation (includes only required packages to speed up env build up)

### CODE REVIEW

File: assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- python=3.10
+- pip=23.1
+- pip:
+  - azureml-mlflow=={{latest-pypi-version}}
+  - azure-ai-ml=={{latest-pypi-version}}
+  - azureml-core=={{latest-pypi-version}}
+  - azureml-telemetry=={{latest-pypi-version}}
+  - mltable=={{latest-pypi-version}}
+  - azureml-automl-core=={{latest-pypi-version}}
+  - azureml-metrics[all]=={{latest-pypi-version}}
```
Comment (ghyadav) [inline assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml:15]: Is this getting used in one of the components?

Comment (ghyadav) [reply assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml:15]: Dependencies that might be needed:
- rouge_score
- nltk

File: assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- python=3.10
+- pip=23.1
+- pip:
+  - azureml-mlflow=={{latest-pypi-version}}
+  - azure-ai-ml=={{latest-pypi-version}}
+  - azureml-core=={{latest-pypi-version}}
+  - azureml-telemetry=={{latest-pypi-version}}
+  - mltable=={{latest-pypi-version}}
+  - azureml-automl-core=={{latest-pypi-version}}
+  - rouge-score
```
Comment (ghyadav) [inline assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml:15]: should we pin these versions?

Comment (iamrk04) [reply assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml]: done, thanks!

**Review by ShipraJain01 — DISMISSED** [review]

File: assets/aml-benchmark/environments/aoai-evaluation/context/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:{{latest-image-tag}}
```
Comment (vizhur) [inline assets/aml-benchmark/environments/aoai-evaluation/context/Dockerfile:1]: you should probably try ubuntu22.04, 20.04 will be deprecated in a few months

File: assets/aml-benchmark/components/batch_output_formatter/spec.yaml [hunk 4]
```diff
@@ -71,7 +71,7 @@ outputs:
   unsafe_content_blocked_requests:
     type: uri_file
 code: ../src
-environment: azureml://registries/azureml/environments/model-evaluation/labels/latest
+environment: azureml://registries/azureml/environments/aoai-evaluation/labels/latest
```
Comment (vizhur) [inline assets/aml-benchmark/components/batch_output_formatter/spec.yaml:74]: I'd expect environment to be released first and tested before updating the components?

**Comment by vizhur** [comment]
@iamrk04 , Rahul, please do not add any labels and follow the instructions in message

File: assets/aml-benchmark/components/batch_output_formatter/spec.yaml (hunk 4 above)
Comment (ShipraJain01) [inline assets/aml-benchmark/components/batch_output_formatter/spec.yaml:74]: we need to change the environment name to 'evaluation'

File: assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+name: aoai-evaluation
```
Comment (ShipraJain01) [inline assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml:1]: we need to change the environment name to 'evaluation'

File: assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+channels:
+- conda-forge
+- anaconda
+dependencies:
+- python=3.10
+- pip=23.1
+- pip:
+  - azureml-mlflow=={{latest-pypi-version}}
+  - azure-ai-ml=={{latest-pypi-version}}
```
[the remark this answers is not in the document]
Comment (ShipraJain01) [reply assets/aml-benchmark/environments/aoai-evaluation/context/conda.yaml:10]: hi @Vitaly,

We would need exception for his one as there is no direct way to currently accommodate Telemetry with only V2 sdk and we need V2 sdk for AI Studio dependencies.
I can take the action item on me to get past this mix of v1 and v2 sdk post ignite.

would really your approvals with exception for this one as its a hard blocker for PuP of AOAI Evals and we have very few days left in Ignite.

File: assets/aml-benchmark/environments/aoai-evaluation/context/Dockerfile (hunk 3 above)
Comment (iamrk04) [reply assets/aml-benchmark/environments/aoai-evaluation/context/Dockerfile]: moved to 22.04

File: assets/aml-benchmark/components/batch_output_formatter/spec.yaml (hunk 4 above)
Comment (iamrk04) [reply assets/aml-benchmark/components/batch_output_formatter/spec.yaml]: removed component update

Comment (iamrk04) [reply assets/aml-benchmark/components/batch_output_formatter/spec.yaml]: removed component update, renamed environment

**Review by vizhur — APPROVED** [review]

**Pull request merged**